BUILD

```diff
@@ -56,6 +56,7 @@ assemble_pip(
         'pyparsing==2.4.2',
         'python-dateutil==2.8.0',
         'pytz==2019.2',
+        'scipy==1.3.1',
```

Comment:
wonder if there is a way to make the BUILD consume the requirements.txt so you don't have to keep them in sync manually

Reply:
I think that's in the pipeline!